src/index.html

```diff
@@ -15,8 +15,100 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="/">
+          <img
+            src="../src/images/logo.png"
+            alt="Site logo"
+          />
+        </a>
+      </div>
+      <nav class="navigation">
+        <ul class="list">
+          <li class="item">
+            <a
+              class="link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="item">
+            <a
+              class="link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="item">
+            <a
+              class="link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="item"
+            data-qa="hover"
+          >
+            <a
+              class="link"
+              href="#"
```

Comment:
Avoid using spaces in the `href` property of the `<a>` tag. Anchor links should start with the `#` symbol. The current value is invalid.